scripts/generate_features.pl

```diff
@@ -36,37 +42,53 @@
 close(FORMAT_FILE);
 
 $/ = $line_separator;
+my %defines_seen;
+my @files = ("$include_dir/config.h");
 
-open(CONFIG_H, "$include_dir/config.h") || die("Failure when opening config.h: $!");
+if( $include_crypto ) {
+    push(@files, "$crypto_include_dir/config.h");
```

Comment:
No, for `config.h`, we should not read both files. `config.h` outside the submodule contains the whole configuration.

Applies to `query_config` as well.

Reply:
If so, then there's no change to do in these files... ?

Reply:
Indeed.